rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/CommandSet.java

```diff
@@ -262,10 +262,10 @@
                                                                           .opt("lj")
                                                                           .longOpt("listjobs")
                                                                           .description("Retrieve a list of jobs managed by the Scheduler.If specified the latest 'x' number of jobs or 'z' maximum number of jobs starting from 'y'th job.") //Example: to return the five last jobs, >./proactive-client -lj limit=5. To return the list of z jobs from index y, >./proactive-client -lj from=y limit=z ")
-                                                                          .numOfArgs(0)
+                                                                          .numOfArgs(1)
```

Comment:
listJobs command has 0 args, and 3 optional arguments. When I set .numOfArgs to 0 (to fix the failing test), the argument "latest", "from" and "limit" are no longer captured.